python/tank/deploy/tank_commands/install.py

```diff
@@ -163,7 +196,8 @@ def _run(self, log, suppress_prompts, env_name, engine_instance_name, app_name):
         log.info("Installing into environment %s and engine %s." % (env_name, engine_instance_name))
     
         try:
-            env = self.tk.pipeline_configuration.get_environment(env_name)
+            env = self.tk.pipeline_configuration.get_environment(env_name, writable=True)
+            env.use_preserving_yaml_parser(preserve_yaml)
```

Comment:
Not a fan of simple getter/setters using another verb than set/get. I was confused at first when I read the method name. Maybe it's just me. I feel set_preserving_yaml_parser is clearer. What do you think?